### PR TITLE
Allow requesting specific gpu in autoscaling e2e tests

### PR DESCRIPTION
**Pull request by losipiuk**
```release-note
NONE
```

### CODE REVIEW

File: test/e2e/autoscaling/cluster_size_autoscaling.go [hunk 1]
```diff
@@ -1530,6 +1536,10 @@ func scheduleGpuPod(f *framework.Framework, id string) error {
 		Labels:         map[string]string{"requires-gpu": "yes"},
 	}
 
+	if gpuType != "" {
+		config.NodeSelector = map[string]string{"cloud.google.com/gke-accelerator": gpuType}
```
Comment (MaciekPytel) [inline test/e2e/autoscaling/cluster_size_autoscaling.go]: Sorry for late comment, but can you make the label name const?

Comment (losipiuk) [reply test/e2e/autoscaling/cluster_size_autoscaling.go]: Done

**Pull request merged**